code/__DEFINES/research/research_categories.dm

```diff
@@ -1,87 +1,186 @@
+#define RND_SUBCATEGORY_MACHINE_ENGINEERING "/Engineering Machines"
+#define RND_SUBCATEGORY_MACHINE_RESEARCH "/Research Machines"
+#define RND_SUBCATEGORY_MACHINE_MEDICAL "/Medical Machines"
+#define RND_SUBCATEGORY_MACHINE_CHEMISTRY "/Chemistry Machines"
+#define RND_SUBCATEGORY_MACHINE_SECURITY "/Security Machines"
+#define RND_SUBCATEGORY_MACHINE_CARGO "/Cargo Machines"
+#define RND_SUBCATEGORY_MACHINE_TELEPORT "/Teleportation Machines"
+#define RND_SUBCATEGORY_MACHINE_ROBOTICS "/Robotics Machines"
+#define RND_SUBCATEGORY_MACHINE_GENETICS "/Genetics Machines"
+#define RND_SUBCATEGORY_MACHINE_FAB "/Fabricators"
+#define RND_SUBCATEGORY_MACHINE_SERVICE "/Service Machines"
+#define RND_SUBCATEGORY_MACHINE_TELECOMMS "/Telecommunications Machines"
+
+#define RND_CATEGORY_EQUIPMENT "/Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_SERVICE "/Service Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_JOKE "/Top-Secret Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_CHEMISTRY "/Chemistry Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_MEDICAL "/Medical Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_SECURITY "/Security Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_TELECOMMS "/Communications Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_BLUESPACE "/Bluespace Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_ENGINEERING "/Engineering Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_ATMOSPHERICS "/Atmospherics Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_SCIENCE "/Science Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_GAS_TANKS "/Gas Tanks"
+#define RND_SUBCATEGORY_EQUIPMENT_XENOBIOLOGY "/Xenobiology Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_GENETICS "/Genetics Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_JANITOR "/Janitorial Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_KITCHEN "/Bar & Kitchen Equipment"
+#define RND_SUBCATEGORY_EQUIPMENT_MISC "/Miscellaneous Equipment"
+
+#define RND_CATEGORY_AI "/Artificial Intelligence"
+#define RND_SUBCATEGORY_AI_CORE "/AI Construction"
+#define RND_SUBCATEGORY_AI_LAW_MANIPULATION "/AI Law Manipulation"
+#define RND_SUBCATEGORY_AI_CORE_MODULES "/Core AI Modules"
+#define RND_SUBCATEGORY_AI_DANGEROUS_MODULES "/Dangerous AI Modules"
+
+#define RND_CATEGORY_MODULAR_COMPUTERS "/Modular Computers"
+#define RND_SUBCATEGORY_MODULAR_COMPUTERS_FRAMES "/Frames"
+#define RND_SUBCATEGORY_MODULAR_COMPUTERS_PARTS "/Parts"
+
+#define RND_CATEGORY_STOCK_PARTS "/Stock Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_MISC "/Miscellaneous Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_TURBINE "/Turbine Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_TELECOMMS "/Telecommunications Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_1 "/Tier 1 Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_2 "/Tier 2 Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_3 "/Tier 3 Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_4 "/Tier 4 Parts"
+#define RND_SUBCATEGORY_STOCK_PARTS_EXCHANGERS "/Exchange Devices"
+
+#define RND_CATEGORY_SYNDICATE "/Syndicate Designs"
```

Comment:
I disagree with it being named Syndicate designs, they are unlocked illegally, but they're more repurposed items. I'd prefer sticking to Illegal Designs, or just keeping it to the original categories they are in (like desynchronizer in bluespace)

Reply:
I agree wholeheartedly :) I'll get this in